docs/index.rst

```diff
@@ -3,6 +3,12 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+.. image:: logo/logo.png
```

Comment:
Looks fine to me. Just can't see the actual output with the logo switching to the markdown preview.

Reply:
The preview for the github-page can be seen here: https://github.com/fabratu/simexpal/tree/20230915_logo